nixos/modules/services/desktops/espanso.nix

```diff
@@ -0,0 +1,40 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.espanso;
+in {
+  options = {
+    services.espanso = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable espanso
+        '';
+      };
+
+      package = mkOption {
```

Comment:
I don't think this option adds much value at this time. Do we need it?

Reply:
It's just something I've got used to do in case you'd like to swap it for unstable if you are running stable. I guess a `packageOverride` or an overlay would do the same thing.